python/pyarrow/array.pxi

```diff
@@ -849,19 +883,48 @@ cdef class Array(_PandasConvertible):
     def sum(self, **kwargs):
         """
         Sum the values in a numerical array.
+
+        See `pyarrow.compute.sum` for full usage.
```

Comment:
We could make this:

```suggestion
        See :func:`pyarrow.compute.sum` for full usage.
```

which would give a proper link in the html version (and not too much visual noise in the text version)

Reply:
Good point, I'll do this.